src/Chapter2.hs

```diff
@@ -327,7 +327,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b xs
+  | a < 0 || b < 0 = []
+  | otherwise = drop a (take (b+1) xs)
```

Comment:
Interesting :thinking: I see, that your implementation works on cases when `b < a`. But it will be more efficient to check this as a separate check, so you can return the empty list immediately.